shared/src/main/scala/com/nawforce/pkgforce/names/Identifier.scala

```diff
@@ -155,23 +169,72 @@ object Identifier {
     Name("upsert"),
     Name("using"),
     Name("virtual"),
+    Name("void"),
     Name("webservice"),
     Name("when"),
     Name("where"),
     Name("while")
   )
 
-  // These are identifiers that don't work but are not reserved, yeah go figure
-  lazy val badIdentifiers: Set[Name] = Set(
+  // Methods can use some identifiers that are reserved
```

Comment:
This would be one of those things where I lean toward being better off taking the strict road and block/dissuade their use anyway even if they can technically be used.

Reply:
We could have a warning on these as a feature enhancement, there are some cases in the apex-samples so I didn't want to error on them